haskell-build/config.yml

```diff
@@ -188,13 +203,15 @@ jobs:
 
           build-steps:
             - build-with-cci-cache:
-                cache-version:          << parameters.cache-version >>
-                cabal-threads:          << parameters.cabal-threads >>
-                cabal-build-extra:      << parameters.cabal-build-extra >>
-                cabal-test-extra:       << parameters.cabal-test-extra >>
-                fail-incoherent-builds: << parameters.fail-incoherent-builds >>
-                run-tests:              << parameters.run-tests >>
-                run-check:              << parameters.run-check >>
-                before-build:           << parameters.before-build >>
-                after-build:            << parameters.after-build >>
-                after-test:             << parameters.after-test >>
+                cache-version:            << parameters.cache-version >>
+                cabal-threads:            << parameters.cabal-threads >>
+                build-component:    << parameters.build-component >>
```

Comment:
nit: odd indentation

Reply:
Indeed, mass-rename (used to be `cabal-build-component`) ;)